liblangutil/Exceptions.h

```diff
@@ -24,33 +24,33 @@
 
 #include <string>
 #include <utility>
+#include <vector>
+#include <memory>
```

Comment:
What includes were hiding these? I see a lot of new system headers introduced in this commit.

Reply:
yes, I wasn't mentioning this in the commit log. sorry. but the if a unit is using a symbol, it should also include the header that is meant to introduce this symbol (I believe I read about that rule in the CppCoreGuidelines, can look for the reference if desired). That's why I was adding them along the way (should be seperate commit/PR).